packages/clerk-js/src/ui/common/Billing.tsx

```diff
@@ -0,0 +1,204 @@
+import type { BillingPlanResource } from '@clerk/types';
+import React from 'react';
+
+import { Badge, Box, Button, Col, descriptors, Flex, Grid, Icon, localizationKeys, Text } from '../customizables';
+import { Header, IconButton } from '../elements';
+import { ArrowLeftIcon, Check } from '../icons';
+
+export type OrganizationPlanCardProps = Pick<BillingPlanResource, 'name' | 'features' | 'priceInCents'> & {
+  isCurrentPlan: boolean;
+  planKey: string;
+  changePlanAction: React.ReactNode;
+};
+
+export const OrganizationPlanCard = (params: OrganizationPlanCardProps) => {
```

Comment:
User and Org plan cards will be identical, right ?

Reply:
Yes they will be identical.